portfolio-website/script.js

```diff
@@ -1,23 +1,79 @@
+// navBar
 const navBarList = [
-    "Ingredients", "Steps to Make", "Links to Other Wonton Soup Recipes", "About Me"
+    "About Me", "Projects", "Contact"
 ]
 
 for (let i = 0; i < navBarList.length; i++) {
     const label = document.createElement('a');
     label.innerText = navBarList[i];
     label.addEventListener('mouseover', function() {
-        label.style.color = "white";
+        label.style.color = "black";
     })
     label.addEventListener('mouseout', function() {
-        label.style.color = "black";
+        label.style.color = "white";
     })
     label.addEventListener('click', function() {
         label.setAttribute('href', `#${navBarList[i].split(" ").join("")}`);
-        if (i === 3) {
-            label.setAttribute('href', "./aboutme.html");
+        if (i === 2) {
+            label.setAttribute('href', "./contactForm.html")
         }
         label.style.textDecoration = "none";
     })
 
     document.getElementById("navBar").appendChild(label);
-}
\ No newline at end of file
+}
+// end navBar
+
+// typewriter functionality
+const span1 = document.createElement('span');
+span1.textContent = "Welcome to";
+const img = document.createElement('img');
+img.setAttribute('src', "https://miro.medium.com/max/875/1*b2ZpVczibdIDlUbuN6S2ZA.png");
+img.setAttribute('id', "pfp");
+const span2 = document.createElement('span');
+span2.textContent = " my website!";
+document.querySelector("#first").append(span1);
+document.querySelector("#last").append(span2);
+let quoteArray = ["Welcome to"];
+let quoteArray2 = ["             my website!"];
```

Comment:
nit: if you can, avoid using 1 or 2 as suffixes. Naming wise would be clearer to reader (without seeing your code) to use quotePrefix, quoteSuffix, textPrefixPosition, textSuffixPosition

 or quoteLeft, quoteRight, textPositionRight, textPositionLeft